eth/ethconfig/gen_config.go

```diff
@@ -24,7 +24,9 @@ func (c Config) MarshalTOML() (interface{}, error) {
 		SnapDiscoveryURLs       []string
 		NoPruning               bool
 		NoPrefetch              bool
-		TxLookupLimit           uint64                 `toml:",omitempty"`
+		TransactionHistory      uint64                 `toml:",omitempty"`
```

Comment:
This is going to be annoying. Flat out changing it will break all configs that people use. We need a way to transition into the new name. We should possibly have both, and if the latter is specified, have it override the former. Then eventually in a long time we can drop it, until then just mark as deprecated. Perhaps have a log message that use XX instead of YY

Reply:
Ah nvm, now I see the old is preserved (or maybe a followup commit added it back)